File: 151001/Aleksandrov/RESTLab1/src/main/java/entities/Issue.java
package entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Timestamp;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Issue {
    Long id;
    Long editorId;
    String title;
    String content;
    Timestamp created;
    Timestamp modified;
}

File: 151001/Kononchuk/REST/REST/Models/Entities/Issue.cs
namespace REST.Models.Entities;

public class Issue
{
    public long Id { get; set; }
    public long EditorId { get; set; } = -1;
    public string Title { get; set; }
    public string? Content { get; set; }
    public DateTime? Created { get; set; }
    public DateTime? Modified { get; set; }
}